### PR TITLE
Make external links to be opened in new browser tab

### PR DESCRIPTION
**Pull request by Hdd87**
installed rehype from https://github.com/rehypejs/rehype-external-links

configured nuxt.config.ts : added markdown.rehypePlugins."rehype-external-links"

### CODE REVIEW

**Comment by d0rich** [comment]
@Hdd87, you are solving business task, so PR should be named as business action

**Review by d0rich — CHANGES_REQUESTED** [review]
Links are still opened in the same tab

File: nuxt.config.ts [hunk 1]
```diff
@@ -40,6 +40,9 @@ export default defineNuxtConfig({
     documentDriven: true,
     highlight: {
       theme: 'one-dark-pro'
+    },
+    markdown: {
+      rehypePlugins: ['rehype-external-links']
```
Comment (d0rich) [inline nuxt.config.ts:45]: Configure target option for external links: https://github.com/rehypejs/rehype-external-links#optionstarget

```suggestion
      rehypePlugins: {
        'rehype-external-links': {
          target: '_blank'
        }
      }
```

**Comment by Hdd87** [comment]
>

**Review by d0rich — APPROVED** [review]

**Comment by Hdd87** [comment]
> Links are still opened in the same tab

sorry didn't know where to put the target: '_blank' ). checked on the local machine links are now opening in a new tab

**Pull request merged**